content/projects/gsoc/2021/projects/cloudevents-plugin.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: gsocproject
+title: "CloudEvents plugin for Jenkins"
+goal: "Be able to listen to and emit cloudevents from Jenkins."
+category: Plugins
+year: 2021
+sig: cloud-native
+status: "Active"
+student: adi10hero
+mentors:
+- "waveywaves"
+- "salaboy"
+- "jeffpearce"
+tags:
+- gsoc2021
+- cloudevents
+- cloudnative
+links:
+  draft: TODO
```

Comment:
draft: https://docs.google.com/document/d/1MvdocqnKLOj2Sf8pqAsd4-GGhd9kl_VXGRPffBFNUOI/edit?usp=sharing

(Shared in edit mode)

Thanks!

Reply:
```suggestion
  draft: https://docs.google.com/document/d/1MvdocqnKLOj2Sf8pqAsd4-GGhd9kl_VXGRPffBFNUOI/edit?usp=sharing
```